### PR TITLE
Docs for 2FA recovery code update

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Updated Docs on how to recover the Docker account when a user has lost the 2FA application and the recovery code.

### CODE REVIEW

File: docker-hub/2fa/recover-hub-account.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ authentication. See [Enabling two-factor authentication on Docker Hub](/docker-h
 ## Alternative recovery methods
 
 If you have lost access to both your two-factor authentication application and
-your recovery code, send an email to [Docker Hub Support](mailto:hub-support@docker.com) from the primary email associated with your Docker ID for recovery instructions.
+your recovery code, get in touch with our Support team using the [Contact Support form](https://hub.docker.com/support/contact/?category=2fa-lockout&username=){:target="_blank" rel="noopener" class="_"}. You must enter the primary email address associated with your Docker ID in the Contact Support form for recovery instructions.
```
Comment (mark-dr) [inline docker-hub/2fa/recover-hub-account.md:41]: We can omit the `username` property from the URL.

```suggestion
your recovery code, get in touch with our Support team using the [Contact Support form](https://hub.docker.com/support/contact/?category=2fa-lockout){:target="_blank" rel="noopener" class="_"}. You must enter the primary email address associated with your Docker ID in the Contact Support form for recovery instructions.
```

**Review by mark-dr — COMMENTED** [review]
LGTM! Only one small suggested change.

**Review by mark-dr — APPROVED** [review]

File: docker-hub/2fa/recover-hub-account.md (hunk 1 above)
Comment (usha-mandya) [reply docker-hub/2fa/recover-hub-account.md]: Thanks @mark-dr. Fixed this.

**Review by FranchommeE — APPROVED** [review]
👍

**Pull request merged**